klasses/models.py

```diff
@@ -0,0 +1,40 @@
+"""Models for klasses"""
+from django.db.models.base import Model
+from django.db.models.fields import (
+    DateTimeField,
+    DecimalField,
+    TextField,
+    IntegerField,
+)
+from django.db.models.fields.related import ForeignKey
```

Comment:
nit: I am pretty sure that you can simply `from django.db import models` and then access everything from there.
`models. Model`, and all the fields are available under `models`